docs/developer/index.rst

```diff
@@ -0,0 +1,19 @@
+Developer
+=========
+
+.. toctree::
+    :titlesonly:
+    
+    contributing
+    architecture
```

Comment:
I see the following drop-down list for System Architecture clause ([link](https://modin--3726.org.readthedocs.build/en/3726/developer/index.html#developer)):
```
Modin PandasDataframe Objects
Generic Ray-based members
PandasOnRay Dataframe implementation
cuDFOnRay Dataframe Implementation
PandasOnDask Dataframe implementation
Experimental
Storage Formats
Query Compiler
PandasOnPython Dataframe implementation
BasePandasDataset
DataFrame Module Overview
Series Module Overview
```
I would prefer to see more high-level components (titles) there.

Reply:
I set the option to show a depth of 4 to increase the visibility of the components in the System Architecture section. Unfortunately that also increases the depth of the other sections, but I think that's okay since the System Architecture section has the most components anyway.